package.json

```
@@ -52,6 +52,9 @@
         "default": "./dist/cjs/locale.js"
       }
     },
+    "./style": {
```

This doesn't sound correct to me: does it mean only Sass can import from `react-day-picker/style`? Why not add a `sass` key to the existing `style.css`? 

Or did you mean:

```suggestion
    "./style.scss": {
```

---

@gpbl 

> does it mean only Sass can import from react-day-picker/style? 

it does mean that at the moment, only Sass can import `react-day-picker/style`. Maybe its a bit odd, but you aren't limited from re-purposing this export in the future. 

> Why not add a sass key to the existing style.css?

This was actually how I first attempted to fix my issue, but like I shared in [my other comment](https://github.com/gpbl/react-day-picker/pull/2555#issuecomment-2437640335), Sass requires that the import URL not end with `.css` when it is meant to be a build-time import. 

> ::about your suggestion::

I'm not sure if that would work. I prefer the extensionless approach because it more closely resembles how a .css file would be treated when using Sass. Adding the `.scss` could work but could be misleading. 🤔